### PR TITLE
Account forAPI  changes in display order of output

### PR DESCRIPTION
**Pull request by DonJayamanne**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
         "theme": "light"
     },
     "engines": {
-        "vscode": "1.54.0-insider"
+        "vscode": "1.55.0-insider"
```
Comment (rchiodo) [inline package.json:27]: Should this have the ```^```?

File: src/client/datascience/notebook/helpers/helpers.ts [hunk 2]
```diff
@@ -346,13 +330,39 @@ function createNotebookCellDataFromCodeCell(cell: nbformat.ICodeCell, cellLangua
...
+    return outputItems.sort((outputItemA, outputItemB) => {
+        const isMimeTypeMatch = (value: string, compareWith: string) => {
+            if (value.endsWith('.*')) {
+                value = value.substr(0, value.indexOf('.*'));
+            }
+            return compareWith.startsWith(value);
+        };
+        const indexOfMimeTypeA = orderOfMimeTypes.findIndex((mime) => isMimeTypeMatch(outputItemA.mime, mime));
+        const indexOfMimeTypeB = orderOfMimeTypes.findIndex((mime) => isMimeTypeMatch(outputItemB.mime, mime));
+        if (indexOfMimeTypeA === indexOfMimeTypeB) {
```
Comment (rchiodo) [inline src/client/datascience/notebook/helpers/helpers.ts:361]: I believe you could just return 'indexOfMimeTypeA' - 'indexOfMimeTypeB'

**Review by rchiodo — APPROVED** [review]

File: src/client/datascience/notebook/helpers/helpers.ts (hunk 2 above)
Comment (DonJayamanne) [reply src/client/datascience/notebook/helpers/helpers.ts:361]: lol, yes

File: package.json (hunk 1 above)
Comment (DonJayamanne) [reply package.json:27]: Not sure we want that,
we want this to be only compatible with 1.55.0-insiders & nothing else.
E.g. if user has `1.55.0` stable, that should not be used.
The API is currently locked into a specific version of VS Code.

Comment (DonJayamanne) [reply package.json:27]: Also we dno't want insider users in `1.56.0-insiders` to get this VSIX.

Comment (DonJayamanne) [reply package.json:27]: Let me know what you think about that.

**Review by DavidKutu — APPROVED** [review]

**Pull request merged**